CHANGELOG.md

```diff
@@ -1,23 +1,48 @@
 #CHANGE LOG
 
+##Version 0.8
+
++ General - project renamed to CloudSlang
++ Tools
+	+ Build Tool - The verifier has been replaced by the build tool, which in addition to checking syntactic validity of a project's files runs the projects associated tests.
++ DSL Additions
+	+ Async Loops - A task can contain a loop which performs an operation for each value in a list in parallel. 
+	+ Tests - Syntax for writing content tests.
```

Comment:
The tests are actually not part of the DSL. I would write it as part of the builder tool